cpplint/cpplint.py

```diff
@@ -1670,7 +1674,7 @@ def GetHeaderGuardCPPVariable(filename):
   fileinfo = FileInfo(filename)
   file_path_from_root = fileinfo.RepositoryName()
   if _root:
-    file_path_from_root = re.sub('^' + _root + os.sep, '', file_path_from_root)
+    file_path_from_root = re.sub('^' + _root + '/', '', file_path_from_root)
```

Comment:
I think this has been merged with a fix for this. PR should be close, I believe (3b0ea894c11140c1, also from 01e47236c84 "444 - Reduced dependency on hardcoded .cc extension.") Also see the fork at https://github.com/theandrewdavis/cpplint for various other fixes around extensions. As an example, the cpplint.py code here still checks for .h extension only in CheckForHeaderGuard, not for hpp and other header extensions used in other places in cpplint.